chromium_src/ui/views/widget/native_widget_mac.h

```diff
@@ -8,8 +8,19 @@
 
 #include "ui/views/widget/native_widget_private.h"
 
-#define SetWindowIcons                    \
-  SetWindowTitleVisibility(bool visible); \
+#define SetWindowIcons                                      \
+  SetWindowTitleVisibility(bool visible);                   \
+  bool has_overridden_window_title_visibility() const {     \
```

Comment:
Does this method actually get used? In |native_widget_mac.mm| it looks like we're directly using the `absl::optional`

Reply:
Yeah, it's used in the test.

Reply:
Could be worth friending the test class? But this is fine :smile:

Reply:
Yeah, sounds good! Only test needs this.

Reply:
Oh, that might bring layering violation as NativeWidgetMac is under ui/views and our test is in chrome layer.